### PR TITLE
Add `@BanyanDB.TimestampColumn` to identify `which column in Record` is providing the timestamp(milliseconds) for BanyanDB

### PR DESCRIPTION
**Pull request by wankai123**
part of https://github.com/apache/skywalking/issues/10008

And polish banyandb Zipkin traces query.
relate to https://github.com/apache/skywalking/pull/10004#discussion_r1029510526

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [X] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Review by mrproliu — APPROVED** [review]

**Review by wu-sheng — DISMISSED** [review]
LGTM. Pending on e2e checking.

**Comment by wankai123** [comment]
Do not merge now, I'm checking.

File: oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBConverter.java [hunk 1]
```diff
@@ -67,9 +67,13 @@ public byte[] getBytes(String fieldName) {
     public static class StreamToStorage implements Convert2Storage<StreamWrite> {
         private final MetadataRegistry.Schema schema;
         private final StreamWrite streamWrite;
+        private final String timestampColumn;
```
Comment (lujiajing1126) [inline oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBConverter.java:70]: I suppose this field can be incorporated into the `MetadataRegistry.Schema` during the model parse, so we don't need this field here.

Comment (wankai123) [reply oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBConverter.java:70]: make sense 👍

**Review by lujiajing1126 — APPROVED** [review]
LGTM

**Pull request merged**